library/azure_rm_virtualmachinescalesetinstance.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_virtualmachinescalesetinstance
+version_added: "2.8"
+short_description: Get Azure Virtual Machine Scale Set Instance facts.
+description:
+    - Get facts of Azure Virtual Machine Scale Set VMs.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
+    vmss_name:
+        description:
+            - The name of the VM scale set.
+        required: True
+    instance_id:
+        description:
+            - The instance ID of the virtual machine.
+        required: True
+    latest_model:
```

Comment:
should call it apply_latest_model?  it's an action..

Reply:
well, it's a works as a state more than an action here. "latest_model" is also a Boolean value in facts.
instances can either be updated to the latest model or not and this state is visible.
so it's not exactly and action like "restart".